SpatialGDK/Source/SpatialGDK/Private/LoadBalancing/WorkingSetSystem.cpp

```diff
@@ -83,6 +83,18 @@ FWorkingSetSystem::FEntityToWorkerMap FWorkingSetSystem::CreateEntityToOwningWor
 
 		const Worker_PartitionId ServerAuthPartitionId = Delegation.Delegations.FindChecked(SpatialConstants::SERVER_AUTH_COMPONENT_SET_ID);
 
+		const EntityViewElement& PartitionEntity = View.FindChecked(ServerAuthPartitionId);
```

Comment:
Let's note that this could be optimized if we did this check after going through partition manager.